code/modules/mob/living/silicon/silicon.dm

```diff
@@ -74,9 +74,11 @@
 /mob/living/silicon/proc/show_laws()
 	return
 
-/mob/living/silicon/drop_item()
-	return
-
+/mob/living/silicon/drop_item(var/Target)
+	var/obj/item/grab/grab = locate() in contents
+	if(grab)
+		return drop_from_inventory(grab)
```

Comment:
Same as previously; isn't there a getter?

Reply:
A getter for what?